specification/synapse/data-plane/Microsoft.Synapse/preview/2019-06-01-preview/gitintegration.json

```diff
@@ -0,0 +1,122 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2019-06-01-preview",
+    "title": "WorkspaceGitRepoManagement"
```

Comment:
change it to SynapseManagementClient

Reply:
It is in data-plane so it should be ArtifactsClient